2-mandatory/6-budgets.js

```diff
@@ -11,301 +11,308 @@ Should give return the answer of 62600.
 **/
 
 function getBudgets(peopleArray) {
+  let sum = 0;
+  for(let i = 0; i < peopleArray.length; i++) {
+    sum = sum + peopleArray[i]
```

Comment:
Almost! Check what ``peopleArray[i]`` actually returns. Also I'd recommend an array method to do this (the for loop above works as well)